### PR TITLE
coadd to spec

### PR DESCRIPTION
**Pull request by rcooke-ast**
A quick update to the `coadd2d` files for PypeIt PR 1036

### CODE REVIEW

**Comment by rcooke-ast** [comment]
Note - this shouldn't be merged until PypeIt `PR 1036` is merged.

**Review by profxj — APPROVED** [review]
ready to merge when 1036 does

**Pull request merged**
